autodist/kernel/synchronization/compressor.py

```diff
@@ -205,6 +205,104 @@ class HorovodCompressorEF(CompressorEF, HorovodCompressor):  # This works becaus
     """Horovod's Compression but with Error Feedback."""
 
 
+class PowerSGDCompressor(CompressorEF):
+    """An implementation of the PowerSGD compression algorithm (arxiv.org/abs/1905.13727)."""
+    def __init__(self, var_op_name, rank=1):
+        self.rank = rank
+        self.og_shape, self.ndims, self.compressor = None, None, None  # compressor is the Q in paper
+        self.var_op_name = var_op_name
```

Comment:
Not needed -- `super.__init__(var_op_name)` sets this field

Reply:
I tried to remove this code. But the prospector told me to add it back to init CompressorEF. So I still keep it.